### DISCUSSION TITLE
LinkFree rebranded to BioDrop

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/LinkFree#8791] [Announcements]
We are excited to announce 🎉... 
- new name **BioDrop**
- new domain [biodrop.io](biodrop.io)
- new brand colours
- new logo

We would love your support if you could **Star** the [repo](https://github.com/EddieHubCommunity/BioDrop) and add your profile to stand out! 🚀 

### Q. What do you think?

<img width="1242" alt="Screenshot 2023-08-26 at 13 03 37" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/4ca1eb71-ea84-4957-88b5-dbf957eff8e1">

**All existing links will still work and will redirect to `biodrop.io`**

*Note: new app design coming soon 💪*

**Comment by SaraJaoude** [comment]
Love the new colours! Can't wait for the new changes which will be coming soon

**Comment by khairnsaa** [comment]
W rebrand. but i think the  letter"B" in "BioDrop" logo is to fat 😂

**Reply by khairnsaa** [reply]
By the way, is there any discord channel for this project? i got assigned to one of the issues and got error while solving it, and i need to ask someone about the errors

**Reply by eddiejaoude** [reply]
Yes, come and chat to us in the EddieHub Discord http://discord.eddiehub.org

**Comment by Pradumnasaraf** [comment]
This is so exciting 🎉. BioDrop for the win.

**Comment by jorgegonzalez** [comment]
Are there any plans to deprecate LinkFree links in the future? Do you plan to keep these working forever?

**Reply by eddiejaoude** [reply]
I don't think so otherwise we will break it for people - I don't see redirecting to be an issue?
